models/incident.js

```diff
@@ -123,25 +123,35 @@ Incident.queryIncidents = function(query, page, options, callback) {
     filter.storedAt.$gte = query.since;
   }
 
-  if (query.veracity == 'confirmed true') filter.veracity = true;
-  if (query.veracity == 'confirmed false') filter.veracity = false;
-  if (query.veracity == 'unconfirmed') filter.veracity = null;
+  if (query.veracity === 'confirmed true') filter.veracity = true;
+  if (query.veracity === 'confirmed false') filter.veracity = false;
+  if (query.veracity === 'unconfirmed') filter.veracity = null;
   if (_.isBoolean(query.veracity)) filter.veracity = query.veracity;
 
-  if (query.status == 'open') filter.closed = false;
-  if (query.status == 'closed') filter.closed = true;
+  if (query.status === 'open') filter.closed = false;
+  if (query.status === 'closed') filter.closed = true;
   delete filter.status;
   if (_.isBoolean(query.closed)) filter.closed = query.closed;
 
-  if (query.escalated == 'escalated') filter.escalated = true;
-  if (query.escalated == 'unescalated') filter.escalated = false;
+  if (query.escalated === 'escalated') filter.escalated = true;
+  if (query.escalated === 'unescalated') filter.escalated = false;
 
   // Search for substrings
   if (query.title) filter.title = new RegExp(query.title, 'i');
   else delete filter.title;
   if (query.locationName) filter.locationName = new RegExp(query.locationName, 'i');
   else delete filter.locationName;
 
+  // Checking for multiple tags in incident
+  if (query.tags) {
+    var tags = [];
+    for (var i = 0; i < query.tags.length; i++) {
+      tags[i] = query.tags[i];
+    }
+    filter.tags = {};
+    filter.tags.$in = tags;
```

Comment:
I would suggest `filter.tags = { $in: query.tags };`, or if it's important to get a copy, `filter.tags = { $in: query.tags.slice() };`.